src/main/kotlin/de/bund/digitalservice/useid/refresh/RefreshController.kt

```diff
@@ -24,21 +26,30 @@ class RefreshController(private val identificationSessionService: Identification
     private val log = KotlinLogging.logger {}
 
     @GetMapping
-    fun redirectToEServiceRefreshAddress(@RequestParam("sessionId") eIDSessionId: UUID, @RequestParam allParams: Map<String, String>): Mono<ResponseEntity<Unit>> {
+    fun redirectToEServiceRefreshAddress(
+        @RequestParam("sessionId") eIDSessionId: UUID,
+        @RequestParam requestQueryParams: Map<String, String>
+    ): Mono<ResponseEntity<Unit>> {
         return identificationSessionService.findByEIDSessionId(eIDSessionId)
             .doOnError {
                 log.error("Failed to load identification session.", it)
             }
             .map {
-                val requestParams = allParams.map { entry -> "${entry.key}=${entry.value}" }.stream().collect(Collectors.joining("&"))
+                val responseQueryParams = buildEncodedQueryParameterString(requestQueryParams)
```

Comment:
Maybe remove the `String` post-fix from the name since we are already expecting that value as String (from the data type)? or maybe we could explicitly add the variable type?
`val responseQueryParams: String = buildEncodedQueryParameter(requestQueryParams)` instead.

Reply:
Done